packages/edit-site/src/components/global-styles/dimensions-panel.js

```diff
@@ -99,6 +120,22 @@ export default function DimensionsPanel( { name } ) {
 		],
 	} );
 
+	const [ contentSizeValue, setContentSizeValue ] = useSetting(
+		'layout.contentSize',
+		name
+	);
+
+	const hasContentSizeValue = () => !! contentSizeValue;
```

Comment:
Taking another look at this check, I'm wondering if part of the issue with the ToolsPanel menu state is that if the `contentSizeValue` matches the value after reset, then this will return `true`. So if my content width is set to `840px` and I change it to `900px` and then reset it, the value here is `840px`, and so the menu item thinks there's still a custom value?

That said, there seems to be something else funny going on, because when I first load the site editor, the ToolsPanel seems to be showing padding as customised, too 🤔

<img width="279" alt="image" src="https://user-images.githubusercontent.com/14988353/179449819-badba077-8775-472e-98f5-6481e7971532.png">

The weird behaviour does appear to be on `trunk`, too, so not sure if it's worth attempting to fix now? I think on `trunk` it's a little less visible because it seems to be working fine at the block level, and we currently only have the `Padding` dimensions control made visible at the root level 🤔

Reply:
Checking for user styles makes sense! Updated.